features/steps/library.py

```
@@ -0,0 +1,18 @@
+
+# -- REGISTER-STEPS FROM STEP-LIBRARY:
+# import behave4cmd0.__all_steps__
+import behave4cmd0.command_steps
+import behave4cmd0.environment_steps
+import behave4cmd0.filesystem_steps
+import behave4cmd0.workdir_steps
+import behave4cmd0.log.steps
+
+# import behave4cmd0.failing_steps
+# import behave4cmd0.passing_steps
+import behave4cmd0.note_steps
```

Ruff (0.8.0)

`behave4cmd0.note_steps` imported but unused

Remove unused import: `behave4cmd0.note_steps`

(F401)